rules/linux/auditd/lnx_auditd_system_info_discovery.yml

```diff
@@ -0,0 +1,30 @@
+title: 'System Information Discovery'
+description: 'Detects System Information Discovery commands'
+author: 'Pawel Mazur'
+status: experimental
+date: 2021/09/03
+references:
+   - 'https://attack.mitre.org/techniques/T1082/'
+   - 'https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1082/T1082.md'
+logsource:
+   product: linux
+   service: auditd
+detection:
+   selection:
+       type: PATH
+       name:
+           - /etc/lsb-release
+           - /etc/redhat-release
+           - /etc/issue
+    condition: 'selection or selection2'
+    selection2:
+       type: EXECVE
+       a0:
+           - uname
+           - uptime
+tags:
```

Comment:
HI
Can you please keep the order logic like the other rule
```yaml
   selection:
       type: PATH
       name:
           - /etc/lsb-release
           - /etc/redhat-release
           - /etc/issue
    selection2:
       type: EXECVE
       a0:
           - uname
           - uptime
    condition: 'selection or selection2'
```

Reply:
Hi,

Corrected